src/scripts/main.js

```diff
@@ -1 +1,70 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(reject, 3000);
+});
+
+firstPromise
+  .then(handleResolve)
+  .catch(handleReject);
+
+const secondPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('Second promise was resolved');
+  });
+
+  document.addEventListener('contextmenu', () => {
+    event.preventDefault();
+    resolve('Second promise was resolved');
+  });
```

Comment:
you have the same resolvers, so create own function at this scope to not repeat yourself. Apply for similar cases

```
const handleResolve() {
      resolve('Second promise was resolved');
    }
```
    
```suggestion
  document.addEventListener('contextmenu', (e) => {
    event.preventDefault();
    handleResolve();
  });
```